frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -44,12 +45,14 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
   canEnablePipelines,
 }) => {
   const [errorMessage, setErrorMessage] = React.useState('');
+
   const {
     dashboardConfig: {
       spec: { notebookController },
     },
   } = React.useContext(AppContext);
   const tolerationSettings = notebookController?.notebookTolerationSettings;
+  const storageClass = usePreferredStorageClass();
```

Comment:
since usePreferredStorageClass() really return one instance of StorageClassKind, maybe be should call this 

`const storageClassPreferred = usePreferredStorageClass();`